core/sagas/inputs.js

```diff
@@ -41,7 +41,9 @@ function flattenInputs(response) {
 function* fetchInputs(action) {
   try {
     const { filter, selectedInputPage, pageSize } = action.payload;
-    const filterValue = `*${filter.value}*`.replace("**", "*");
+    const wildcardsUsed = filter.value.includes("*");
+    let filterValue = filter.value.replace(/, /g, ",").replace(/ /g, ",");
```

Comment:
I believe you can use the regex or operator `|` here to create only one regex pattern.

```const regex = / |, /g```

We might also want to specify that these replacements should come between words with `\b`. So we don't add a comma at the beginning.

```const regex = /\b \b|\b, \b/g```

Also it might be a good idea to replace multiple strings using so `a,  b` doesn't get converted to `*a*,**,*b*`.

```const regex = /\b +\b|\b, +\b/g```

Reply:
Thanks for the suggestions! 

If I use `\b` and the filter value includes wildcards, like `yell* pink *purple*`, then these separate values do not meet the criteria for `\b` and therefore the spaces are not replaced.

For the case where the user starts or ends with a space, the only alternative I can think of is to look for and remove any comma we inserted from the beginning or end of the string. Do you have any alternate suggestions?

Reply:
I included updates using the first `const regex = / |, /g` example you suggested along with ` +` in one commit, and then an example of what I suggested above for removing commas from the start or end in the next commit. Let me know if you have any other suggestions I should try instead.